package.json

```diff
@@ -23,7 +23,8 @@
     "test:coverage": "npm test -- --coverage --watchAll=false",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
     "eject": "react-scripts eject",
-    "snyk-protect": "snyk protect"
+    "snyk-protect": "snyk protect",
+    "prepare": "npm run snyk-protect"
```

Comment:
@snyk-bot I know I'm writing to a bot but I swear to god if you don't either 

1. Stop adding these two lines every time you make a PR, or
2. Fix the bug in these tasks that throws a JS memory error and causes the build to fail

then I will walk over to the Snyk offices myself and pour water down your server rack.